src/components/TextButton/TextButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { Text, TouchableWithoutFeedback, View } from "react-native";
+import styles from "./styles";
+
+const TextButton = ({ text, addOns, isDisabled = false, styles: customStyles, onPress }: TextButtonProps) => {
+  return (
+    <TouchableWithoutFeedback
+      onPress={onPress}
+      disabled={isDisabled}
+      {...addOns}
+    >
+      <View>
+        <Text style={[styles.text, customStyles, isDisabled && styles.isDisabled]}>{text}</Text>
```

Comment:
use prettier

Reply:
My prettier print width is set to 125
so this line is under 125 character


will apply ternary in this
`isDisabled && styles.isDisabled`